cache/cache_test.cc

```diff
@@ -492,8 +499,9 @@ TEST_P(CacheTest, EvictionPolicyRef) {
   Insert(302, 103);
   Insert(303, 104);
 
-  // Insert entries much more than Cache capacity
-  for (int i = 0; i < kCacheSize * 2; i++) {
+  // Insert entries much more than cache capacity.
+  double load_factor = std::min(fast_lru_cache::kLoadFactor, clock_cache::kLoadFactor);
+  for (int i = 0; i < static_cast<int>(kCacheSize / load_factor); i++) {
```

Comment:
This is because with clock, unlike LRU, even after inserting 2x capacity, some old element can get evicted after the newer ones if we get unlucky.